src/main/java/io/cryostat/net/OpenShiftAuthManager.java

```diff
@@ -287,6 +345,54 @@ private String getTokenFromHttpHeader(String rawHttpHeader) {
         }
     }
 
+    private Future<URI> computeAuthorizationEndpoint() {
+
+        return CompletableFuture.supplyAsync(
+                () -> {
+                    CompletableFuture<JsonObject> oauthMetadata = new CompletableFuture<>();
+                    try {
+                        String namespace = this.getNamespace();
+                        String clientId =
+                                String.format(
+                                        "system:serviceaccount:%s:%s",
+                                        namespace, env.getEnv("CRYOSTAT_OAUTH_CLIENT_ID"));
+
+                        String scope =
+                                String.format(
+                                        "user:check-access role:%s:%s",
+                                        env.getEnv("CRYOSTAT_OAUTH_ROLE"), namespace);
```

Comment:
Can we add some guards in case the env vars are not set/blank? It looks to me like the failure mode here would be a little tough to figure out just looking at logs if one of these env vars wasn't set. Better to fail earlier and with a very clear error message in case the OpenShift platform is selected and the prerequisite environment variables appear to be missing.